.travis.yml

```diff
@@ -156,31 +156,23 @@ matrix:
       osx_image: xcode9.3
       env:
         - DESC="MacOS JDK8 verify, site, assembly
-        - CMD1="mvn -e package -Dlinkcheck.skip=true"
-        - CMD2="mvn -e package -Passembly"
-        - CMD="export JAVA_HOME=$(/usr/libexec/java_home) && $CMD1 && $CMD2"
+        - CMD="./.ci/travis/travis.sh travis-osx"
 
     # MacOS JDK11 verify (till cache is not working, we can not do verify)
     - os: osx
       osx_image: xcode10.1
       env:
         - DESC="MacOS JDK11 verify, site, assembly
-        - CMD1="mvn -e package -Dlinkcheck.skip=true"
-        - CMD2="mvn -e package -Passembly"
-        - CMD="export JAVA_HOME=$(/usr/libexec/java_home) && $CMD1 && $CMD2"
+        - CMD="./.ci/travis/travis.sh travis-osx"
```

Comment:
Please call it as:
./.ci/travis/travis.sh osx-package
./.ci/travis/travis.sh osx-assembly

Reply:
Done